kotlin/codegen/build.gradle.kts

```diff
@@ -44,6 +44,7 @@ dependencies {
     exclude(group = "org.jetbrains.kotlin", module = "kotlin-stdlib")
   }
   implementation(Dependencies.KotlinPoet.kotlinPoet)
+  kapt(Dependencies.KotlinPoet.kotlinPoet)
```

Comment:
Why is this needed? The only processors on the processorpath are auto service and incap helper.

Reply:
Actually seems it's not needed. As said in both issues this is not a part I usually use and master.
During first attempt adding this did embed kotlinpoet in the jar so I assumed it was necessary and part of the fix.

Just tested and just properly binding the shadowJar task at proper timing is enough. PR updated.